<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageTaskWindow</class>
 <widget class="QMainWindow" name="ImageTaskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QSplitter::handle
{
    background-color: rgba(255, 255, 255, 80);
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="imageViewContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="imageViewContainerLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="geometrize::dialog::ScriptConsole" name="consoleWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string extracomment="The title of the File menu that contains commands relating to the handling of files, such as open, save, print, etc">File</string>
    </property>
    <addaction name="actionLoad_Settings_Template"/>
    <addaction name="actionSave_Settings_Template"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string extracomment="The title of the View menu that contains commands relating to the handling of visual elements, such as the size and visibility of windows and dialogs within the app">View</string>
    </property>
    <addaction name="actionReveal_Launch_Window"/>
    <addaction name="actionScript_Console"/>
    <addaction name="actionPixmap_Results_View"/>
    <addaction name="actionVector_Results_View"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QDockWidget" name="statsDock">
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>800</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string extracomment="Title of a section that contains statistics about a task, such as the length of time a task has been running">Stats</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="geometrize::dialog::ImageTaskStatsWidget" name="statsDockContents"/>
  </widget>
  <widget class="QDockWidget" name="imageSettingsDock">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>1350</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string extracomment="Title of a section that contains settings that affect the image that the app will transform into shapes">Image Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="targetImageSettingsDockContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="targetImageSettingsScrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="geometrize::dialog::ImageTaskImageWidget" name="imageTaskImageWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>282</width>
          <height>660</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="exporterDock">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>1350</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string extracomment="Title of a section that contains preferences and settings for and saving and exporting results data and images">Exporters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="exporterDockContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QScrollArea" name="exporterScrollArea">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="geometrize::dialog::ImageTaskExportWidget" name="imageTaskExportWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>332</width>
          <height>263</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="runnerSettingsDock">
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>1350</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string extracomment="Title of a section that contains preferences and settings that affect the image geometrizing task">Task Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="runnerSettingsDockContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QScrollArea" name="runnerSettingsScrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="geometrize::dialog::ImageTaskRunnerWidget" name="imageTaskRunnerWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>332</width>
          <height>263</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="scriptsDock">
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>2400</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Scripts</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="scriptsDockContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QScrollArea" name="scriptsDockScrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="geometrize::dialog::ImageTaskScriptingWidget" name="scriptsWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>332</width>
          <height>263</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string extracomment="Text on a menu item that closes the window when pressed">Close</string>
   </property>
  </action>
  <action name="actionSave_Settings_Template">
   <property name="text">
    <string extracomment="Text on a menu item that opens a file saving dialog, allowing the user to select a location to save a settings file">Save Settings File</string>
   </property>
  </action>
  <action name="actionReveal_Launch_Window">
   <property name="text">
    <string extracomment="Text on a menu item that opens or reveals the main launch window of the app when pressed">Show Launch Window</string>
   </property>
  </action>
  <action name="actionLoad_Settings_Template">
   <property name="text">
    <string extracomment="Text on a menu item that opens a file selection dialog, allowing the user to select a settings file to load">Load Settings File</string>
   </property>
  </action>
  <action name="actionScript_Console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string extracomment="Text on a menu item that shows or hides the script console when pressed">Script Console</string>
   </property>
  </action>
  <action name="actionReveal_Script_Editor">
   <property name="text">
    <string extracomment="Text on a menu item that opens or reveals the script editor panel when pressed">Reveal Script Editor</string>
   </property>
  </action>
  <action name="actionPixmap_Results_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string extracomment="Text on a menu item that shows or hides a window containing a pixmap/bitmap/raster drawing of the shapes the app has created">Pixmap Image View</string>
   </property>
  </action>
  <action name="actionVector_Results_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string extracomment="Text on a menu item that shows or hides a window containing a vector-based (SVG) graphic of the shapes the app has created">Vector Graphics View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>geometrize::dialog::ScriptConsole</class>
   <extends>QWidget</extends>
   <header>dialog/scriptconsole.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>geometrize::dialog::ImageTaskStatsWidget</class>
   <extends>QWidget</extends>
   <header>dialog/imagetaskstatswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>geometrize::dialog::ImageTaskImageWidget</class>
   <extends>QWidget</extends>
   <header>dialog/imagetaskimagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>geometrize::dialog::ImageTaskExportWidget</class>
   <extends>QWidget</extends>
   <header>dialog/imagetaskexportwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>geometrize::dialog::ImageTaskRunnerWidget</class>
   <extends>QWidget</extends>
   <header>dialog/imagetaskrunnerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>geometrize::dialog::ImageTaskScriptingWidget</class>
   <extends>QWidget</extends>
   <header>dialog/imagetaskscriptingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
